bazel/envoy_select.bzl

```diff
@@ -32,6 +32,21 @@ def envoy_select_hot_restart(xs, repository = ""):
         "//conditions:default": xs,
     })
 
+# Selects the given values depending on the WASM runtimes enabled in the current build.
+def envoy_select_wasm_wavm(xs):
```

Comment:
You could just use `envoy_select_wasm_wavm_or(xs, [])` instead.

Reply:
done